javascript-data-model/order-history.js

```diff
@@ -0,0 +1,79 @@
+var orderHistory = [
+  {
+    itemInfo: {
+      typeOfItem: 'Book',
+      nameOfItem: 'JavaScript for impatient programmers',
+      author: 'Rauschmayer, Dr. Axel',
+      itemPrice: 31.55,
+      itemImage: 'rhino-book-image.jpg',
+      returnWindowCloses: 'September 7, 2020'
+    },
+    orderPlaced: 'August 4, 2020',
+    deliveredDate: 'August 8, 2020',
+    orderTotal: 34.00,
+    shipTo: 'JS Masher',
+    orderNum: '114-3941689-8772232',
+    deliveredStatus: true,
+    handedToResident: false
+  },
+  {
+    itemInfo: {
+      typeOfItem: 'Book',
+      nameOfItem: 'The Timeless Way of Building',
+      author: 'Alexander, Christopher',
+      itemPrice: 41.33,
+      itemImage: 'timeless-way.jpg',
+      returnWindowCloses: 'August 19, 2020'
+    },
+    orderPlaced: 'July 19, 2020',
+    deliveredDate: 'July 20, 2020',
+    orderTotal: 44.53,
+    shipTo: 'JS Masher',
+    orderNum: '113-9984268-1280257',
+    deliveredStatus: true,
+    handedToResident: true
+  },
+  {
+    itemInfo: {
+      typeOfItem: 'Hardware',
+      nameOfItem: 'Gamecube Controller Adapter. Super Smash Bros Switch Gamecube Adapter for Wii U, PC. Support Turbo and Vibration Features. No Driver and No Lag-Gamecube Adapter',
+      itemPrice: 15.98,
+      itemImage: 'gamecube-adaptor.jpg',
+      returnWindowCloses: 'August 5, 2020'
+    },
+    orderPlaced: 'July 4, 2020',
+    deliveredDate: 'July 7, 2020',
+    orderTotal: 17.22,
+    shipTo: 'JS Masher',
+    orderNum: '114-2875557-9059409',
+    deliveredStatus: true,
+    handedToResident: true
+  },
+  {
+    itemInfo:
+    [
+      {
+        typeOfItem: 'Hardware',
+        nameOfItem: 'GameCube Controller - Super Smash Bros. Edition (Nintendo Switch)',
+        itemPrice: 94.95,
+        itemImage: 'gamecube-controller.jpg',
+        returnWindowCloses: 'September 7, 2020'
+      },
+      {
+        typeOfItem: 'Book',
+        nameOfItem: 'The Art of Sql',
+        author: 'Faroult, Stephane',
+        itemPrice: 33.99,
+        itemImage: 'art-sql.jpg',
+        returnWindowCloses: 'September 7, 2020'
+      }
+    ],
```

Comment:
For this order, the value of this property is an array, but for all your other orders, the value is an object. This is inconsistent and could lead to bugs.